### DISCUSSION TITLE
how to set node edge line type with script?

### DISCUSSION THREAD

**Discussion by ghost** [freeplane/freeplane#1809] [Help] [answered]
![image](https://github.com/freeplane/freeplane/assets/153435789/4163b8cd-a2ac-488a-8571-79fbbce11771)

**Comment by ghost** [comment]
and connector line type
![image](https://github.com/freeplane/freeplane/assets/153435789/83441122-d79b-4659-bda2-cee187bc1213)

**Comment by euu2021** [comment, marked as answer]
See the Reference Guide: https://docs.freeplane.org/scripting/Reference.html

```
// Sets "CLOSE_DOTS" as the edge line type.

node.style.edge.dash = "CLOSE_DOTS"
```

```
// Sets the dash array pattern of the outside connector located at index 0 of the selected node.

node.connectorsOut[0].dashArray = "CLOSE_DOTS"
```

**Reply by ghost** [reply]
Thank you, The document is written in a very friendly manner. Wish you have a good day.
